sale_stock_picking_invoicing/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module extends Stock Picking Invoicing implementation to Sale, you can define the 'Sale Invoicing Policy', if the invoice should be created from Sale Order or from Stock Picking, in this case, the information used at invoice come from Sale Order e.g.: Price Unit, Payment Terms, and others fields.
```

Comment:
This is missing the one of the functionalities of this module : for stock.moves, override price calculation that is present in stock_picking_invoicing, with the native Sale Order Line price calculation, same for the partner_id

Reply:
I tried to include your description, let me know if now it's better